_assets/scss/modules/_projects.scss

```diff
@@ -45,6 +65,13 @@
   }
 }
 
+.more-projects {
+  .h2 {
+    line-height: 1em;
```

Comment:
Since "More projects" will always be on a single line, I'm removing the extra vertical space caused by the line-height to avoid increasing discrepancies with the baseline grid (something we should talk about soon)